src/styles/main.scss

```
@@ -1,3 +1,71 @@
+$cell-color: #eee;
+$cell-side: 100px;
+$cell-border-color: black;
+$days-count: 31;
+$cells-gap: 1px;
+$days-list: mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7;
+$cell-hover-color: pink;
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
+
+.calendar {
+  margin: 0;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cells-gap;
+  width: 7 * $cell-side + 6px + 20px; /* 7 = days count, 6 = gap's count, 20px = 2*10px padding */
+
+  padding: 10px;
+
+  list-style: none;
+
+  &__day {
+    position: relative;
+    width: $cell-side;
+    height: $cell-side;
+    box-sizing: border-box;
+
+    background-color: $cell-color;
+    border: 1px solid $cell-border-color;
+
+    font-size: 30px;
+
+    cursor: pointer;
+
+    transition: 0.5s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```